Foodfeedback/Services/Loginservice.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Foodfeedback.DTO;
+using Foodfeedback.Models;
+using Foodfeedback.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System.Linq;
+
+namespace Foodfeedback.Services
+{
+    public class Loginservice : ILoginService
+    {
+        bool ILoginService.Userdetails([FromBody]UserDTO userDTO)
```

Comment:
Change the method name to match the functionality eg. CheckIfUserExists() or ValidateUser()